### DISCUSSION TITLE
Greenland landcover type is not showing up after running LDT (both IGBPNCEP and USGS).

### DISCUSSION THREAD

**Discussion by jayaisro** [NASA-LIS/LISF#1608] [Q&A] [answered]
Hello LISF community!

I have just started using LIS, I wanted to run LIS for snow water equivalent and snow cover simulations over Greenland using the Noah MP 3.6 model. After I run LDT for the lis_input generation, i find that the landmask is correctly generated but the landcover has all values of 0/1 over Greenland.

I have tried with both USGS and NCEP modified IGBP (20 classes) option. I get the same issue.

I have added the ldt.config and ldtlog at the bottom along with some screenshots of landmask, USGS landcover and IGBP NCEP landcover. 

I would really appreciate if anyone could help provide some info/support on this issue.

Land Mask:
![image](https://github.com/user-attachments/assets/753141fb-a3c4-49a1-9496-56f3df926a71)

USGS land cover:
![image](https://github.com/user-attachments/assets/372148e0-cb6c-41c1-81fa-7d4649fff0dd)

IGBP NCEP land cover:
![image](https://github.com/user-attachments/assets/b8e417e3-2d73-41d2-a809-178a6c839613)

#%% config entries for USGS:
LDT running mode:             "LSM parameter processing"  # LDT type of run-mode (top-level option)
Processed LSM parameter filename:  ./lis_input.d01.nc   # Final output file read by LIS-7

LIS number of nests:                   1         # 1 nest supported for now
Number of surface model types:         1
Surface model types:                  LSM        # LSM | Lake
Land surface model:                "Noah-MP.3.6"      # Enter LSM of choice
Routing model:                       none
Lake model:                          none      # Lake model option: 'none'|'FLake'
#Include water points:               .false.      # Include water points
Water fraction cutoff value:          0.5
Incorporate crop information:       .false.

Number of met forcing sources:         1                # Enter number of forcing types
Met forcing sources:                 GDAS             # Enter 'none' if no forcing selected
Met spatial transform methods:      bilinear            # bilinear | budget-bilinear | neighbor | average
Topographic correction method (met forcing):  lapse-rate    # none | lapse-rate

LDT diagnostic file:              ldtlog                # Log-based diagnostic output file
Undefined value:                   -9999.0              # Universal undefined value
LDT output directory:               OUTPUT              # If metrics or stats are written out
Number of ensembles per tile:         1                 # The number of ensemble members per tile

# LIS domain:
Map projection of the LIS domain:             latlon
Run domain lower left lat:                    58.00
Run domain lower left lon:                    -75.00
Run domain upper right lat:                   85.00
Run domain upper right lon:                   -09.00
Run domain resolution (dx):                   0.05
Run domain resolution (dy):                   0.05

Output methodology:                           "2d gridspace"
Output data format:                           netcdf

Noah-MP PBL Height Value:                     900.

#Land Parameters
Landcover data source:                        "USGS_Native"
Landcover classification:                     "USGS"
Landcover file:                               /projects/nrsc04/models/lis_input/lis_param/ldt_param/veg30susgs
Landcover spatial transform:                  neighbor
Landcover map projection:                     latlon
Landcover fill option:                        neighbor                  # none | neighbor (Not needed if creating landmask)
Landcover fill value:                         10
Landcover fill radius:                        5

Create or readin landmask:                    create
Landmask data source:                         MODIS_Native
Landmask spatial transform:                   mode

#Soil texture map:
Soil texture data source:                     STATSGOFAO_Native
Soil texture map:                             /projects/nrsc04/models/lis_input/lis_param/ldt_param/STATSGO/topsoil30snew  # Enter soil texture map
Soil texture spatial transform:               neighbor              # none | mode | neighbor | tile
Soil texture fill option:                     neighbor                # none | neighbor
Soil texture fill radius:                     4                     # Number of pixels to search for neighbor
Soil texture fill value:                      5                     # Static value to fill where missing 
Soil texture map projection:     latlon
Soil texture lower left lat:    -59.9950
Soil texture lower left lon:   -179.9950
Soil texture upper right lat:    89.9950
Soil texture upper right lon:   179.9950
Soil texture resolution (dx):     0.01
Soil texture resolution (dy):     0.01

Soils map projection:          latlon
Soils spatial transform:       none   # Note: do not use mode with soil fractions
Soils fill option:             none

#elevation
Elevation data source:                        GTOPO30_Native
Elevation number of bands:                    1
Elevation map:                                /projects/nrsc04/models/lis_input/lis_param/ldt_param/GTOPO30/fetch_old
Elevation fill option:            neighbor
Elevation fill radius:            5
Elevation fill value:             100
Elevation number of bands:        1
Topography spatial transform:     average
Topography map projection:       latlon
Topography lower left lat:      -59.995
Topography lower left lon:     -179.995
Topography upper right lat:      89.995
Topography upper right lon:     179.995
Topography resolution (dx):       0.01
Topography resolution (dy):       0.01

# Albedo maps
Albedo data source:                           NCEP_Native
Albedo map:                                   /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo       # Albedo files
Albedo climatology interval:    monthly
Albedo spatial transform:       average #budget-bilinear
Albedo fill option:             neighbor       # none, neighbor, average
Albedo fill radius:             5
Albedo fill value:              0.1
Albedo map projection:          latlon
Albedo lower left lat:         -59.99500
Albedo lower left lon:        -179.99500
Albedo upper right lat:         89.99500
Albedo upper right lon:        179.99500
Albedo resolution (dx):          0.0100
Albedo resolution (dy):          0.0100

# Snow albedo maps
Max snow albedo data source:                  NCEP_Native
Max snow albedo map:                          /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/maxsnoalb.asc  # Max. snow albedo map
Max snow albedo spatial transform:     average # budget-bilinear
Max snow albedo fill option:           neighbor        # none, neighbor, average
Max snow albedo fill radius:          5
Max snow albedo fill value:           0.3
Max snow albedo map projection:        latlon
Max snow albedo lower left lat:      -59.99500
Max snow albedo lower left lon:     -179.99500
Max snow albedo upper right lat:      89.99500
Max snow albedo upper right lon:     179.99500
Max snow albedo resolution (dx):       0.0100
Max snow albedo resolution (dy):       0.0100

# Greenness fraction maps
Greenness data source:                        NCEP_Native
Greenness fraction map:                       /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac          # Greenness fraction map        
Greenness climatology interval:               monthly 
Greenness maximum map:                        /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_max.asc  # Maximum greenness fraction map
Greenness maximum fill value:     1.0
Greenness minimum map:                        /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_min.asc  # Minimum greenness fraction map
Greenness minimum fill value:     0.0
Greenness spatial transform:       average #budget-bilinear
Greenness fill option:             neighbor
Greenness fill radius:             5
Greenness fill value:              0.20
Greenness map projection:         latlon
Greenness lower left lat:       -59.9950
Greenness lower left lon:      -179.9950
Greenness upper right lat:       89.9950
Greenness upper right lon:      179.9950
Greenness resolution (dx):        0.010
Greenness resolution (dy):        0.010
Calculate min-max greenness fraction: .false.

# Slope type map
Slope type data source:                       NCEP_Native
Slope type map:                               /projects/nrsc04/models/lis_input/lis_param/ldt_param/param/islope         # Slope type map
Slope type spatial transform:   neighbor
Slope type fill option:         neighbor        # none, neighbor, average
Slope type fill radius:         5
Slope type fill value:          2
Slope type map projection:     latlon
Slope type lower left lat:    -59.99500
Slope type lower left lon:    -179.9950
Slope type upper right lat:    89.99500
Slope type upper right lon:   179.99500
Slope type resolution (dx):     0.0100
Slope type resolution (dy):     0.0100

# Bottom temperature map
Bottom temperature data source:               ISLSCP1
Bottom temperature map:                       /projects/nrsc04/models/lis_input/lis_param/ldt_param/param/SOILTEMP.60     # Bottom soil temperature file
Bottom temperature spatial transform:    budget-bilinear
Bottom temperature topographic downscaling:  "lapse-rate"  # none | lapse-rate
Bottom temperature fill option:      average       # none, neighbor, average
Bottom temperature fill value:       287.0       # Static value to fill where missing
Bottom temperature fill radius:       15.        # Number of pixels to search for neighbor
Bottom temperature map projection:      latlon
Bottom temperature lower left lat:     -59.9950
Bottom temperature lower left lon:    -179.9950
Bottom temperature upper right lat:     89.9950
Bottom temperature upper right lon:    179.9950
Bottom temperature resolution (dx):      0.010
Bottom temperature resolution (dy):      0.010

#------------------------------------------------------
#Forcing parameters
#------------------------------------------------------

GDAS T126 elevation map: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t126.grb
GDAS T170 elevation map: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t170.grb
GDAS T254 elevation map: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t254.grb
GDAS T382 elevation map: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t382.grb
GDAS T574 elevation map: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t574.grb
GDAS T1534 elevation map: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography_uf.t1534.3072.1536.grb

#%% LDT LOG
 ----------------------------------
  Start of LDT parameter processing 
 ----------------------------------
 local domain:(        1321         541 )
 local domain without halo:(        1321         541 )
 running domain:(        1321         541 )
 local domain:    58.0000000       85.0000000      -75.0000000      -8.99999905    
 -------------------- LDT/LIS Domain ----------------------
 (           1 ,   0.00000000     )
 (           2 ,   1321.00000     )
 (           3 ,   541.000000     )
 (           4 ,   58.0000000     )
 (           5 ,  -75.0000000     )
 (           6 ,   128.000000     )
 (           7 ,   85.0000000     )
 (           8 ,  -8.99999905     )
 (           9 ,   5.00000007E-02 )
 (          10 ,   5.00000007E-02 )
 (          11 ,   64.0000000     )
 (          12 ,   0.00000000     )
 (          13 ,   0.00000000     )
 ----------------------------------------------------------
 LSM User-selected:  Noah-MP.3.6
  - - - - - - MODEL PARAMETERS (NOT Selected) - - - - - - - 
 Regional mask data source: not defined
 WARNING: Soil fraction data source: not defined
 Slope data source: not defined
 Aspect data source: not defined
 PPT climatology data source: not defined
 Tmin climatology data source: not defined
 Tmax climatology data source: not defined
  -- Opening LDT Mask-Parameter Fill log file -- 
 [INFO]  - - - - - - -  SURFACE TYPES  - - - - - - - - 
 [INFO] Initializing and Summing Surface Types 
 [INFO] ... Surface types selected: 
  ... LSM       
 [INFO] Finished initializing surface types
  [INFO] - - - - - - - - - Landcover/Landmask Parameters - - - - - - - - - - -
 [INFO] Parameter-Mask Agreement Selected for Landcover (neighbor)
 Param_Check: Neighboring vegetation type being selected ...
 [INFO] Reading landcover values
 [INFO] Reading USGS-Native landcover file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/veg30susgs
 #Regridding: Applying nearest neighbor to input parameter
 [INFO] LAT/LON -- Creating mask output
 [INFO] Checking/filling mask values for: LANDCOVER
 [INFO] SURFACETYPE -- Assigning landcover surface types
 [INFO] Finished assigning LSM surface types
 [INFO] Finished reading landmask and landcover data
  - - - - - - - - - - Glacier Parameters - - - - - - - - - - - - - -
  - - - - - - - - - - Soils Parameters - - - - - - - - - - - - - -
 Param_Check: Nearest neighbor of soil texture class being selected ...
 [INFO] Reading Native STATSGO+FAO texture file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/STATSGO/topsoil30snew
 #Regridding: Applying nearest neighbor to input parameter
 [INFO] Done reading Native STATSGO+FAO texture file
 Checking/filling mask values for: TEXTURE
  - - - - - - - - - Greenness Fraction Parameters - - - - - - - - -
 Param_Check: Average greenness fraction values to be calculated ...
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_jan.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_jan.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_feb.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_feb.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_mar.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_mar.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_apr.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_apr.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_may.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_may.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_jun.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_jun.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_jul.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_jul.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_aug.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_aug.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_sep.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_sep.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_oct.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_oct.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_nov.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_nov.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_dec.asc
 [INFO] Reading NCEP Native Monthly Greenness 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_dec.asc
 Checking/filling mask values for: GREENNESS
 [INFO] Reading NCEP Native greenness max file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_max.asc
 #Regridding: Applying average to input parameter
 [INFO] Done reading NCEP Native greenness max file.
 Checking/filling mask values for: SHDMAX
 [INFO] Reading NCEP Native greenness min file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/gfrac/gfrac_min.asc
 #Regridding: Applying average to input parameter
 [INFO] Done reading NCEP Native greenness min file. 
 Checking/filling mask values for: SHDMIN
  - - - - - - - - - - Albedo Parameters - - - - - - - - - - - - -
 Param_Check: Average max snow albedo values to be calculated ...
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/maxsnoalb.asc
 [INFO] Reading NCEP Native max snow albedo file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/maxsnoalb.asc
 #Regridding: Applying average to input parameter
 [INFO] Done reading native max snow alb file
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/maxsnoalb.asc
 Checking/filling mask values for: MXSNALBEDO
 Param_Check: Average albedo values to be calculated ...
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_jan.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_jan.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_feb.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_feb.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_mar.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_mar.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_apr.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_apr.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_may.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_may.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_jun.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_jun.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_jul.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_jul.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_aug.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_aug.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_sep.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_sep.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_oct.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_oct.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_nov.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_nov.asc
 Reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_dec.asc
 [INFO] Reading NCEP Native Monthly Albedo 
 #Regridding: Applying average to input parameter
 Done reading /projects/nrsc04/models/lis_input/lis_param/ldt_param/albedo/albedo_dec.asc
 Checking/filling mask values for: ALBEDO
  - - - - - - - - - Topographic Parameters - - - - - - - - - - - -
 Param_Check: Average topographic values to be calculated ...
 [INFO] Reading Native GTOPO30 elevation files found in directory: /projects/nrsc04/models/lis_input/lis_param/ldt_param/GTOPO30/fetch_old
 #Regridding: Applying average to input parameter
 [INFO] Done reading Native GTOPO30 elevation files.
 Checking/filling mask values for: ELEVATION
  - - - - - - - - - Slope Type Parameter - - - - - - - - - - - -
 Param_Check: Estimate neareast neighbor slope type values ...
 [INFO] Reading NCEP Native slopetype file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/param/islope
 #Regridding: Applying nearest neighbor to input parameter
 [INFO] Done reading native slope type file
 Checking/filling mask values for: SLOPETYPE
  - - - - - - - - - Bottom Temperature Parameter - - - - - - - - - - - -
 Param_Check: Estimate conservative interpolated bottom temperature values ...
 [INFO] Reading ISCLSP1 (1.0 deg res) Native Bottom Temperature file: /projects/nrsc04/models/lis_input/lis_param/ldt_param/param/SOILTEMP.60
 #Regridding: Applying budget-bilinear interp to input parameter
 Checking/filling mask values for: TBOT
 Performing lapse-rate correction to Tbot output.
  - - - - - - - - - Noah-MP Parameters - - - - - - - - - - - -
  - - - - - - - - Meteorological Parameters - - - - - - - -
 Reading the GDAS forcing data
 [INFO] Initializing GDAS grid-1 for: 1991-2000 grid (T126)
 [INFO] Initializing GDAS grid-2 for: 2000-2002 grid (T170)
 [INFO] Initializing GDAS grid-3 for: 2002-2005 grid (T254)
 [INFO] Initializing GDAS grid-4 for: 2005-2010 grid (T382)
 [INFO] Initializing GDAS grid-5 for: 2010-2015 grid (T574)
 [INFO] Initializing GDAS grid-6 for: 2015-present grid (T1534)
  Projection and Transform: gaussian, bilinear
 Reading forcing elev for GDAS_T126
 Opening/reading GDAS elev file:: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t126.grb
 #Regridding: Applying bilinear interp to input parameter
 Done reading forcing elevation field. 
  Projection and Transform: gaussian, bilinear
 Reading forcing elev for GDAS_T170
 Opening/reading GDAS elev file:: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t170.grb
 #Regridding: Applying bilinear interp to input parameter
 Done reading forcing elevation field. 
  Projection and Transform: gaussian, bilinear
 Reading forcing elev for GDAS_T254
 Opening/reading GDAS elev file:: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t254.grb
 #Regridding: Applying bilinear interp to input parameter
 Done reading forcing elevation field. 
  Projection and Transform: gaussian, bilinear
 Reading forcing elev for GDAS_T382
 Opening/reading GDAS elev file:: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t382.grb
 #Regridding: Applying bilinear interp to input parameter
 Done reading forcing elevation field. 
  Projection and Transform: gaussian, bilinear
 Reading forcing elev for GDAS_T574
 Opening/reading GDAS elev file:: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography.t574.grb
 #Regridding: Applying bilinear interp to input parameter
 Done reading forcing elevation field. 
  Projection and Transform: gaussian, bilinear
 Reading forcing elev for GDAS_T1534
 Opening/reading GDAS elev file:: /projects/nrsc04/models/lis_input/lis_param/elevation/25KM/global_orography_uf.t1534.3072.1536.grb
 #Regridding: Applying bilinear interp to input parameter
 Done reading forcing elevation field. 
 -- Finished writing parameters to netcdf output file -- 
 --------------------------------
  Finished LDT run 
 --------------------------------

**Comment by dmocko** [comment, marked as answer]
Hi @jayaisro 

Thanks for your interest in LIS.

I will take a closer look at your ldt.config and ldtlog files later today, but I think I know why you are seeing this behavior.

The "LANDCOVER" field in the lis_input field is a layered field, where each layer is the fraction of that grid point that is
that particular landcover classification.  The values are not either 0 or 1, but can be any fraction from (and including) 0.0 to 1.0.

If you use "ncdump" on the lis_input file, you can see that this field is also dimensioned by the number of landcover types:

```
        float LANDCOVER(sfctypes, north_south, east_west) ;
                LANDCOVER:standard_name = "MODIS-IGBP (NCEP-modified) landcover map" ;
                LANDCOVER:units = "" ;
                LANDCOVER:scale_factor = 1.f ;
                LANDCOVER:add_offset = 0.f ;
                LANDCOVER:missing_value = -9999.f ;
                LANDCOVER:vmin = 0.f ;
                LANDCOVER:vmax = 0.f ;
                LANDCOVER:num_bins = 20 ;
```
where:

```
dimensions:
        east_west = 11700 ;
        north_south = 6500 ;
        sfctypes = 20 ;
```

You are only looking at the first layer of the "LANDCOVER" field in the plots that you have attached.

If you use "ncview" on the lis_input file, you can change the "sfctypes" value from 1 to 20 to see the fractions of
each of the landcover types.  And if you sum up these 20 layers, it will total to 1.0.  It will total to 1.0 for even the
water points, as water is one of the landcover types in the IGBP landcover classification.

Please let us know if you would like further explanation.

**Reply by jayaisro** [reply]
Hi @dmocko,

Thankyou! what you have mentioned is correct, the lis_input file created is correct, I completely missed the 3th dimension, a grave oversight, my apologies on it. 

I appreciate the help provided!
